packages/core/state/interaction/selectors.ts

```diff
@@ -119,6 +122,24 @@ export const getFileService = createSelector(
     }
 );
 
+/**
+ * Selector specifically for open-source dataset manifest, re-using regular file service interface.
+ * Used in web only
+ */
+export const getPublicDatasetManifestService = createSelector(
+    [getDatasetManifestSource, getPlatformDependentServices],
+    (datasetManifestSource, platformDependentServices): FileService | undefined => {
+        if (!datasetManifestSource || !platformDependentServices) {
+            return undefined;
+        }
+        return new DatabaseFileService({
```

Comment:
Is this any different than the normal DatabaseFileService returned from the `getFileService` selector?

Reply:
The only difference is the dependence on `dataManifestSource`/`getDatasetManifestSource` instead of on `dataSource`/`getDataSource`. I considered reusing the same dataSource state, but was concerned about the side effects of having them both pointing to the same thing (e.g., would it cause things to re-render in app that shouldn't)

Reply:
Also, `getFileService` defaults to `httpFileService` if no data source is provided, which I don't think we want to do here. This one returns null if there's no dataset manifest